src/Hedgehog.Xunit/PropertyAttribute.fs

```diff
@@ -133,7 +132,14 @@ type {t.Name} =
     | :? IDisposable as d -> d.Dispose()
     | _ -> ()
 
+  let resultIsOk r =
+    match r with
+    | Ok _ -> true
+    | Error _ -> false
```

Comment:
Put this in a (probably internal) module called `Result` and call the function `isResult`.